src/ci/docker/host-x86_64/mingw-check/Dockerfile

```
@@ -36,6 +36,7 @@ ENV SCRIPT python3 ../x.py --stage 2 test src/tools/expand-yaml-anchors && \
            python3 ../x.py build --stage 0 src/tools/build-manifest && \
            python3 ../x.py test --stage 0 src/tools/compiletest && \
            python3 ../x.py test --stage 2 src/tools/tidy && \
+           python3 ../x.py test --stage 0 library/std && \
```

This just tests the std crate and not any others. Can you also test the other crates?
```suggestion
           python3 ../x.py test --stage 0 core alloc std test proc_macro && \
```